moodlecli/main.py

```diff
@@ -275,6 +277,8 @@ def export_bulk(input_csv, bucket_name, directory, data_type):
             aws.put_json_data(grade_data, bucket_name, key)
         elif data_type == 'users':
             user_data = moodle.get_users_by_course(id)
+            uuid_data = moodle.get_user_uuids([])
```

Comment:
I would pull this out of the loop so you get the data once per invocation. Currently you're making this same query for every course :grimacing: .